uiowa/uiowa
# Events by presenter listing

Resolves #5968 

# How to test

Event feature:
```
ddev blt ds && ddev drush @default.local config-split:activate event -y && ddev drush @default.local uli people/herky-hawk
```
- View Herky's page to ensure that no view display shows up.
- Go to https://default.uiowa.ddev.site/node/add/event
- Add an event called "Parent event" and add Herky as a presenter.
- Add another event called "Child event" and add Herky as a presenter.
- Add an event in the future and in the past.
- If you want to add a total of 12 events to make sure that no more than 10 show up, I won't stop you.
- View Herky's page again: https://default.uiowa.ddev.site/people/herky-hawk
- Check that the event listing meets the criteria in #5968
- Ensure that "Parent event" doesn't show up on Herky's page

ISWF:
```
ddev blt ds --site iowasummerwritingfestival.uiowa.edu && ddev drush @iowasummerwritingfestival.local config-split:activate event -y && ddev drush @iowasummerwritingfestival.local en better_exposed_filters -y && ddev drush @iowasummerwritingfestival.local config:import --source ../config/sites/iowasummerwritingfestival.uiowa.edu --partial -y && ddev drush @iowasummerwritingfestival.local cim -y 
```
- Repeat similar testing as above.

Admissions:
```
ddev blt ds --site admissions.uiowa.edu && ddev drush @admissions.local uli counselor/jessica-klein
```
- Just ensure that the present events block does not display on the person page.
